paz/pipelines/keypoints.py

```diff
@@ -14,6 +14,11 @@
 from ..backend.keypoints import flip_keypoints_left_right, uv_to_vu
 from ..datasets import JOINT_CONFIG, FLIP_CONFIG
 
+import numpy as np
+from paz.processors.keypoints import SimpleBaselines3D
+from paz.datasets.human36m import data_mean2D, data_stdev2D, data_mean3D,\
+    data_stdev3D, dim_to_use3D
+
```

Comment:
Please don't import directly from PAZ from "..".
we should remove the use of "\" just have the import in one line

Reply:
resolved